frontend/src/views/HomeView.vue

```diff
@@ -1 +1,58 @@
-<template><h1>Test</h1></template>
+<script lang="ts" setup>
+import { onMounted, ref } from 'vue'
+import { ElButton, ElDialog, ElInput } from 'element-plus'
+import { useRolesStore } from '@/stores/roles/index'
+import { ErrorMessage } from '@/services/messages/messages'
+import { SuccessMessage } from '@/services/messages/messages'
+
+const dialogFormVisible = ref(false)
+const HTTP_STATUS_CODE_CONFLICT = 409
+const rolesStore = useRolesStore()
+const newRoleName = ref('')
+
+onMounted(async () => {
+  await rolesStore.fetchRoles()
+})
+
+const onClickCreateRole = () => {
+  dialogFormVisible.value = true
+}
+
+const onClickCancel = () => {
+  dialogFormVisible.value = false
+}
+
+const createRole = async () => {
+  if (newRoleName.value == '') {
+    ErrorMessage('You need to fill in this field')
```

Comment:
You can return after this message. This way, you won't need the else statement

Reply:
If you don't return in this case, the function will try to create the role, even showing the message. Have you tested this?